test/plugin/scenarios/shardingsphere-5.0.0-beta-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,183 @@
+      startTime: not null
+      endTime: not null
+      componentId: 32
+      isError: false
+      spanType: Exit
+      peer: localhost:-1
+      skipAnalysis: false
+      tags:
+      - {key: db.type, value: sql}
+      - {key: db.instance, value: demo_ds_0}
+      - {key: db.statement, value: SELECT * FROM t_order_1}
+    - operationName: /ShardingSphere/executeSQL/
+      operationId: 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Unknown
+      startTime: not null
+      endTime: not null
+      componentId: 60
+      isError: false
+      spanType: Local
+      peer: ''
+      skipAnalysis: false
+      refs:
+      - {parentEndpoint: /shardingsphere-5.0.0-beta-scenario/case/execute, networkAddress: '',
+        refType: CrossThread, parentSpanId: 2, parentTraceSegmentId: not null,
+        parentServiceInstance: not null, parentService: shardingsphere-5.0.0-beta-scenario,
+        traceId: not null}
+  - segmentId: not null
+    spans:
+    - operationName: H2/JDBI/PreparedStatement/executeQuery
+      operationId: 0
+      parentSpanId: 0
+      spanId: 1
+      spanLayer: Database
+      startTime: not null
+      endTime: not null
+      componentId: 32
+      isError: false
+      spanType: Exit
+      peer: localhost:-1
+      skipAnalysis: false
+      tags:
+      - {key: db.type, value: sql}
+      - {key: db.instance, value: demo_ds_1}
+      - {key: db.statement, value: SELECT * FROM t_order_1}
+    - operationName: /ShardingSphere/executeSQL/
+      operationId: 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Unknown
+      startTime: not null
+      endTime: not null
+      componentId: 60
+      isError: false
+      spanType: Local
+      peer: ''
+      skipAnalysis: false
+      refs:
+      - {parentEndpoint: /shardingsphere-5.0.0-beta-scenario/case/execute, networkAddress: '',
+        refType: CrossThread, parentSpanId: 2, parentTraceSegmentId: not null,
+        parentServiceInstance: not null, parentService: shardingsphere-5.0.0-beta-scenario,
+        traceId: not null}
+  - segmentId: not null
+    spans:
+    - operationName: /ShardingSphere/parseSQL/
+      operationId: 0
+      parentSpanId: 0
+      spanId: 1
+      spanLayer: Unknown
+      startTime: not null
+      endTime: not null
+      componentId: 60
+      isError: false
+      spanType: Local
+      peer: ''
+      skipAnalysis: false
+      tags:
+      - {key: db.statement, value: SELECT * FROM t_order}
+    - {operationName: /ShardingSphere/routeSQL/, operationId: 0, parentSpanId: 2,
+      spanId: 3, spanLayer: Unknown, startTime: not null, endTime: not null,
+      componentId: 60, isError: false, spanType: Local, peer: '', skipAnalysis: false}
+    - {operationName: /ShardingSphere/rewriteSQL/, operationId: 0, parentSpanId: 2,
+      spanId: 4, spanLayer: Unknown, startTime: not null, endTime: not null,
+      componentId: 60, isError: false, spanType: Local, peer: '', skipAnalysis: false}
```

Comment:
These spans are not well formated, and more below, please fix

Reply:
Fixed.